velox/type/TimestampConversion.cpp

```diff
@@ -171,6 +171,7 @@ bool tryParseDateString(
   int32_t month = -1;
   int32_t year = 0;
   bool yearneg = false;
+  int signSize = 0;
```

Comment:
Perhaps use a bool flag as this value can only be 0 or 1.

Reply:
What I want is to go directly to `pos - signSize` without judgment.

Reply:
Bool can also do that as the conversion to int is implicit.